django/cantusdb_project/main_app/views/chant.py

```diff
@@ -776,56 +798,76 @@ def get_queryset(self) -> QuerySet:
                 # as a substring
                 feasts = Feast.objects.filter(name__icontains=feast)
                 q_obj_filter &= Q(feast__in=feasts)
-            if self.request.GET.get('order'):
-                if self.request.GET.get('order') == 'siglum':
-                    order = 'siglum'
-                elif self.request.GET.get('order') == 'incipit':
+            order: str = 'siglum'
```

Comment:
I don't think this is used?

Reply:
with [1ae779a](https://github.com/DDMAL/CantusDB/pull/568/commits/1ae779a97e266e0276e57a3a6ebe36f5a2611d29), this has been replaced by an `else` statement.